packages/@aws-cdk/aws-certificatemanager/lambda-packages/dns_validated_certificate_handler/lib/index.js

```diff
@@ -97,6 +98,17 @@ const requestCertificate = async function(requestId, domainName, subjectAlternat
 
   console.log(`Certificate ARN: ${reqCertResponse.CertificateArn}`);
 
+
+  if (!!tags) {
+    const map = new Map(Object.entries(tags));
+    const result = Array.from(map).map(([Key, Value]) => ({ Key, Value }))
```

Comment:
I believe the `new Map` here is redundant, unless there's a specific (edge) case I'm missing. The below should work:

```suggestion
    const result = Array.from(Object.entries(tags)).map(([Key, Value]) => ({ Key, Value }))
```